src/TConsole.cpp

```diff
@@ -635,7 +639,7 @@ void TConsole::resizeEvent(QResizeEvent* event)
 
     QWidget::resizeEvent(event);
 
-    if (!mIsDebugConsole && !mIsSubConsole) {
+    if (mType & (MainConsole|Buffer)) {
```

Comment:
We do not need a `WindowResizeEvent` for the `Buffer` type but since it is not visible we probably have not been seeing the code being run for them!